package/config_types/config_list.js

```diff
@@ -0,0 +1,79 @@
+/**
+  * @class
+  * @extends { Array }
+*/
+class ConfigList extends Array {
+  get(key) {
+    const index = this.getIndex(key, true)
+    return this[index].value
+  }
+
+  set(key, value) {
+    return this.add({ key, value })
+  }
```

Comment:
I think we should deprecate `set` in favor of `append` since `append` / `prepend` have nice symmetry. For now, let's have `set` call `append` instead of the other way around.

Reply:
Sure, will make a PR